README.md

```diff
@@ -38,6 +38,10 @@
 
 Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. [Get started →.](#getting-started)
 
+<img src="images/semgrep-main-image.jpg" />
```

Comment:
@rajaraodv Can you remove the drop shadow and curvature on this image? I think it would look better if it’s a plain screenshot, and the same for other images further down in the README.

Also, can you change the paths the image sources to match the convention used by the logo on line 3? This is needed because the README is automatically displayed in other places (like PyPI) and those need an absolute URL to render the image.

Reply:
Got it!